orttraining/orttraining/core/graph/optimizer/adam_optimizer_builder.h

```diff
@@ -11,7 +11,12 @@ namespace training {
 class AdamOptimizerBuilder final : public OptimizerBuilder {
  public:
   AdamOptimizerBuilder() : OptimizerBuilder("AdamOptimizer",
-                                            {"alpha", "beta", "lambda", "epsilon", "do_bias_correction"}) {}
+                                            {"alpha",
+                                             "beta",
+                                             "lambda",
+                                             "epsilon",
+                                             "do_bias_correction",
+                                             "weight_update_mode"}) {}
```

Comment:
>weight_update_mode [](start = 46, length = 18)

weight_update_mode or weight_decay_mode?

Reply:
weight_decay_mode, was a typo. the new iteration has the correct name